nixos/modules/system/boot/stage-1.nix

```diff
@@ -274,6 +274,8 @@ let
       ${pkgs.buildPackages.busybox}/bin/ash -n $target
     '';
 
+    verbose = config.boot.initrd.verbose;
```

Comment:
Why not add this to the `inherit (config.boot.initrd)` line below?

Reply:
Because I did not pay a close enough attention to the code style^^" Should be fixed now.